packages/moonstone/internal/Picker/PickerButton.js

```diff
@@ -88,7 +88,15 @@ const PickerButtonBase = kind({
 			);
 		} else {
 			return (
-				<IconButton {...rest} backgroundOpacity="transparent" disabled={disabled} small>
+				<IconButton
+					{...rest}
+					backgroundOpacity="transparent"
+					disabled={disabled}
+					holdConfig={{events: [
```

Comment:
Should we externalize this to a module-scoped variable to avoid generating a new instance every render? Should it also be applied to the `joined` version as well?

Reply:
Seems reasonable to apply to `joined` as well so the mouse or arrow key behavior is similar in both configurations.

Reply:
The performance issue does not replicate with the `joined` version of `Picker`, so a fix there isn't required. However, I'm ok with keeping the hold behavior consistent across `Picker` variations, even though it's inconsistent across components. I don't have strong feelings either way.